app/src/main/java/com/kickstarter/viewmodels/inputs/CreatorDashboardHeaderHolderViewModelInputs.java

```diff
@@ -0,0 +1,10 @@
+package com.kickstarter.viewmodels.inputs;
+
+import com.kickstarter.models.Project;
+import com.kickstarter.models.ProjectStats;
+
+
+public interface CreatorDashboardHeaderHolderViewModelInputs {
```

Comment:
I believe @luoser has been moving to a new pattern of nesting the inputs and outputs inside the viewmodel, e.g.: https://github.com/kickstarter/android-oss/blob/master/app/src/main/java/com/kickstarter/viewmodels/MessageHolderViewModel.java#L19-L48